src/System.Private.CoreLib/shared/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -295,7 +294,7 @@ private static unsafe void CopyToManaged<T>(IntPtr source, T[] destination, int
 
             // The rest of the argument validation is done by CopyTo
 
-            new Span<T>((void*)source, length).CopyTo(new Span<T>(destination, startIndex, length));
+            new Span<T>((void*)source, length).CopyTo(destination.AsSpan(startIndex, length));
```

Comment:
Same question... they're functionally identical, as AsSpan just calls the ctor, but using AsSpan just forces the JIT to do more inlining.  Is there a benefit?